bytecomp/bytepackager.mli

```diff
@@ -20,16 +20,10 @@ val package_files:
   ppf_dump:Format.formatter -> Env.t -> string list -> string -> unit
 
 type error =
-    Forward_reference of string * Cmo_format.compunit
-  | Multiple_definition of string * Cmo_format.compunit
+    Forward_reference of string * Compilation_unit.t
+  | Multiple_definition of string * Compilation_unit.t
   | Not_an_object_file of string
-<<<<<<< HEAD
-  | Illegal_renaming of Compilation_unit.Name.t * string * string
-||||||| 121bedcfd2
-  | Illegal_renaming of string * string * string
-=======
-  | Illegal_renaming of Cmo_format.compunit * string * Cmo_format.compunit
->>>>>>> 5.2.0
+  | Illegal_renaming of Compilation_unit.Name.t * string * Compilation_unit.t
```

Comment:
I suspect what we'll land on here is `Compilation_unit.t * string * Compilation_unit.t` but it's hard to say without seeing the changes we need to make to `bytepackager.ml`.

Reply:
OK I've changed this.